mm2src/coins/utxo/utxo_common.rs

```diff
@@ -752,6 +752,34 @@ pub fn tx_size_in_v_bytes(from_addr_format: &UtxoAddressFormat, tx: &UtxoTx) ->
     }
 }
 
+/// Implements building utxo script pubkey for an address by the address format
+pub fn get_script_for_address(coin: &UtxoCoinFields, addr: &Address) -> Script {
+    match addr.addr_format {
+        UtxoAddressFormat::Standard => {
+            if addr.prefix == coin.conf.pub_addr_prefix && addr.t_addr_prefix == coin.conf.pub_t_addr_prefix {
+                return Builder::build_p2pkh(&addr.hash);
+            } else if addr.prefix == coin.conf.p2sh_addr_prefix && addr.t_addr_prefix == coin.conf.p2sh_t_addr_prefix {
+                return Builder::build_p2sh(&addr.hash);
+            }
```

Comment:
Maybe we should return an error here if neither of the if conditions are true, just to be safe.